### DISCUSSION TITLE
Backfill more history graphs with statistics data

### DISCUSSION THREAD

**Discussion by MakeItWorkNow** [home-assistant/frontend#19760] [Dashboard]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

Entities Card - Header & Footer Graph Feature Cannot Display more than 10 days of data.

### Describe the behavior you expected

With the new updates and long term statistics now available in HA, the Entities Card - Header & Footer Graphs should be able to access data ever 10 days(240 hrs).

### Steps to reproduce the issue

1. go to main dashboard
2. Select hamburger menu in top right corner, then edit dashboard
3. Select [Add Card] button at bottom right
4. Search for and select Entities card
5. Select plus sign (+) beside Header/ Footer
6. Select [Graph] in Choose a Header/ Footer section
7. Select an entity for graph & input hours to show
8. …..Add other required info etc. and save
...¡

### What version of Home Assistant Core has the issue?

Home Assistant 2024.1.6

### In which browser are you experiencing the issue with?

Multiple

### Which operating system are you using to run this browser?

Windows 11

**Comment by ildar170975** [comment]
> With the new updates and long term statistics now available in HA, the Entities Card - Header & Footer Graphs should be able to access data ever 10 days(240 hrs).

Only a History view contains added statistical data.
History-graph card, Sensor card - still show only historical (not LTS) data.

By a design, not a bug.

**Comment by MakeItWorkNow** [comment]
Well, since the data is already available  for history view and long term stats, then the design should be updated so this data can be used in the graphs.

**Comment by ildar170975** [comment]
Feature requests are proposed in another section.
You can propose this in Discussions or in Community "Feature requests".
